src/style-stars--3.css

```diff
@@ -0,0 +1,3 @@
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
```

Comment:
It seems like you're using a relative path for the background image, which could potentially lead to issues if the CSS file is moved or the directory structure changes. It's a good practice to use absolute paths or to define a base path variable for assets to ensure they're always correctly located.